### PR TITLE
Fix issues for rhel6 test.

### PR DESCRIPTION
**Pull request by rbian**
Attach-interface operation don't support "--live" option,
Error messages differ from rhel7 to rhel6.

### CODE REVIEW

File: libvirt/tests/src/virtual_network/iface_hotplug.py [hunk 1]
```diff
@@ -76,8 +76,14 @@ def create_iface_xml(mac):
                                               flagstr=attach_option,
                                               ignore_status=True)
                     if ret.exit_status:
-                        if ret.stderr.count("No more available PCI slots"):
+                        if (ret.stderr.count("No more available PCI slots") or
+                                ret.stderr.count("No more available"
+                                                " PCI addresses")):
```
Comment (will-Do) [inline libvirt/tests/src/virtual_network/iface_hotplug.py]: I suggest that use a err_msg list here, so we can append the list if any more error need append.

File: libvirt/tests/src/virtual_network/iface_hotplug.py [hunk 2]
```diff
@@ -76,8 +76,14 @@ def create_iface_xml(mac):
...
                             break
+                        elif (ret.stderr.count("doesn't support option %s"
+                                               % attach_option)):
+                            raise errorTestNAError("Not supported option %s"
+                                                   % attach_option)
```
Comment (will-Do) [inline libvirt/tests/src/virtual_network/iface_hotplug.py]: Just need raise ```ret.stderr`` here

**Comment by will-Do** [comment]
BTW, pls fix the Travis CI complain.

**Comment by rbian** [comment]
Thanks @will-Do , fixed.

**Comment by Hao-Liu** [comment]
@rbian, It seems the current `break` can only break out the internal for-loop, which is not expected result.   Please help check this again. Thanks.

**Comment by rbian** [comment]
@Hao-Liu , Yes, that's a problem.

File: libvirt/tests/src/virtual_network/iface_hotplug.py [hunk 3]
```diff
@@ -76,8 +78,16 @@ def create_iface_xml(mac):
                                               flagstr=attach_option,
                                               ignore_status=True)
                     if ret.exit_status:
-                        if ret.stderr.count("No more available PCI slots"):
+                        find_msg = False
+                        for msg in err_msgs:
+                            if ret.stderr.count(msg):
+                                find_msg = True
+                                break
+                        if find_msg:
```
Comment (Hao-Liu) [inline libvirt/tests/src/virtual_network/iface_hotplug.py]: @rbian, How about use `any` to make it cleaner. like:
```
if any([msg in ret.stderr for msg in err_msgs]):
    log something
    break
```

Comment (rbian) [inline libvirt/tests/src/virtual_network/iface_hotplug.py]: That's great! I will modify the code.

**Comment by Hao-Liu** [comment]
@rbian, Now this PR LGTM. please test it and we can merge this.

**Comment by will-Do** [comment]
ACK and merge this PR

**Pull request merged**